js/data.js

```diff
@@ -1381,7 +1426,7 @@ var data = {
         kennung: [ITSE, BPET, OSIS, SAMT],
         lehrform: ["Vorlesung"],
         modul: ["IT-Systems Engineering", "Vertiefungsgebiete"],
-        semester: [ws13_14, ws14_15],
+        semester: [ws13_14, ws14_15, ws15_16],
```

Comment:
Same as above. Can't use this old lecture, because now it offers 6 LP and the specialication areas are different.